### DISCUSSION TITLE
How to add line breaks?

### DISCUSSION THREAD

**Discussion by 97carmine** [eta-dev/eta#125] [Q&A]
With express.js I send a .txt file through a template that is received correctly, the problem is that it does not add the line break automatically when the variable is declared in the end of the line.

Template syntax that adds the line break:

<details>
<summary>Option 1: Convert to string with \n:</summary>

```
Developer: <%= `${it.developer}\n` %>
contact: <%= `${it.contact}\n` %>
```

</details>
<details>
<summary>Option 2: Add a space after the closing tag:</summary>

```
Developer: <%= it.developer %> 
contact: <%= it.contact %> 
```

</details>

Result:
```
Developer: Name
contact: email
```

---

Template syntax not adding line break:
```
Developer: <%= it.developer %>
contact: <%= it.contact %>
```

Result:
```
Developer: Namecontact: email
```

**Comment by CyberSeppi** [comment]
same here, any hint ?
